workloads/files/workload-namespaces-per-cluster-script-cm.yml

```diff
@@ -80,7 +80,8 @@ data:
       result_dir=${benchmark_results_dir}
     fi
     start_time=$(date +%s)
-    VIPERCONFIG=/tmp/namespaces_per_cluster.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the cluster [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
+    git clone https://github.com/cloud-bulldozer/snafu; pip install -r snafu/requirements.txt;
```

Comment:
ditto

Reply:
nice suggestion, https://github.com/openshift-scale/images/pull/25 has been opened for this. Once that's merged I will update this pr and remove the extra step of installing requirements.